src/ui/tray-icon.cpp

```diff
@@ -660,7 +660,11 @@ void SeafileTrayIcon::deleteAccount()
         return;
     }
 
+#ifdef Q_OS_WIN32
     QString question = tr("Are you sure to remove account from \"%1\"?").arg(account.serverUrl.toString());
+#else
+    QString question = tr("Are you sure to remove account from \"%1\"? You can find files that are downloaded yet at ~/Library/CloudStorage.").arg(account.serverUrl.toString());
```

Comment:
After removing account, you can find still find downloaded files at ~/Library/CloudStorage.